src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -92,6 +92,18 @@
     /**
      * A list of statements.
      *
+     * <p>For example:</p>
+     * <pre>
+     * {
```

Comment:
@SGanguly1999 thanks for your PR
it may be difficult for users to understand token usages from this example, 
try to use example with method definition or constructor definition

Reply:
Sure @timurt